### PR TITLE
Add original protobuf lib files.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

**Comment by floitsch** [comment]
Almost direct copy of the original files. (I only changed one import from absolute to relative).
In this stage doesn't work as it uses primitives that don't exist anymore.

File: src/protobuf.toit [hunk 1]
```diff
@@ -0,0 +1,570 @@
+// Copyright (C) 2020 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the lib/LICENSE file.
```
Comment (erikcorry) [inline src/protobuf.toit:3]: LICENSE file location seems wrong.

File: src/protobuf.toit [hunk 2]
```diff
@@ -0,0 +1,570 @@
...
+  write_map key_type/int value_type/int map/Map --as_field/int?=null --oneof/bool=false [serialize_key] [serialize_value] -> none
+  write_message_header msg/Message --as_field/int?=null --oneof/bool=false -> none
+  reset -> none
+
+  constructor out/bytes.Buffer:
+    return Writer_ out
+
+abstract class Message:
+  // TODO: serialize should be abstract but default values
+  // to flags are not allowed for abstract methods
```
Comment (erikcorry) [inline src/protobuf.toit:56]: Still true?

**Review by erikcorry — APPROVED** [review]

File: src/protobuf.toit (hunk 2 above)
Comment (floitsch) [reply src/protobuf.toit:56]: Will look into it.

**Pull request merged**